src/hardware/targets.json

```diff
@@ -487,6 +487,14 @@
                 "firmware": "Unified_ESP8285_2400_RX",
                 "prior_target_name": "DIY_2400_RX_PWMPEX"
             },
+            "pwm-vario": {
+                "product_name": "Generic ESP32 PWM Vario 2.4Ghz RX",
+                "lua_name": "Vario 2400RX",
+                "layout_file": "Vario 2400.json",
+                "upload_methods": ["uart", "wifi", "betaflight"],
```

Comment:
Is betaflight passthrough really relevant for this one?

Reply:
It isn't strictly, but Betaflight Passthrough is just "via_UART except for an up-and-running receiver". I left it in for when the Outputs can be configured as CRSF TX/RX eventually, but I can remove it for now and add it back when that feature is available.